Update debian release

Compiling a rust program that depends on the `ring` crate fails with the following error:

`/home/cross/project/target/arm-unknown-linux-gnueabihf/release/deps/libring-2ca5101b10054897.rlib(ring-2ca5101b10054897.ring.cklkti8e-cgu.4.rcgu.o): In function `spin::once::Once$LT$T$GT$::call_once::h2af17645f2763809':
          ring.cklkti8e-cgu.4:(.text._ZN4spin4once13Once$LT$T$GT$9call_once17h2af17645f2763809E+0x74): undefined reference to `getauxval'
          ring.cklkti8e-cgu.4:(.text._ZN4spin4once13Once$LT$T$GT$9call_once17h2af17645f2763809E+0x84): undefined reference to `getauxval'
          collect2: error: ld returned 1 exit status`

I am not sure 100% what the issue is but on my Raspberry Pi (with /etc/debian_version -> 10) the `getauxval ` symbol seems to be defined in `./arm-linux-gnueabihf/libc-2.28.so`. When I run a shell in the container it looks like it's using `libc-2.13.so`.

Any thoughts?

--- a/bin/gcc-sysroot
+++ b/bin/gcc-sysroot
@@ -1,2 +1,2 @@
 #!/bin/bash
-arm-linux-gnueabihf-gcc --sysroot=$HOME/pi-tools/arm-bcm2708/arm-bcm2708hardfp-linux-gnueabi/arm-bcm2708hardfp-linux-gnueabi/sysroot "$@"
+arm-linux-gnueabihf-gcc --sysroot=$HOME/pi-tools/arm-bcm2708/arm-rpi-4.9.3-linux-gnueabihf/arm-linux-gnueabihf/sysroot "$@"

--- a/bin/run.sh
+++ b/bin/run.sh
@@ -23,7 +23,8 @@ fi
 
 #Include the cross compilation binaries
 export PATH=$TOOLCHAIN:$PATH;
-export SYSROOT="$HOME/pi-tools/arm-bcm2708/arm-bcm2708hardfp-linux-gnueabi/arm-bcm2708hardfp-linux-gnueabi/sysroot";
+export SYSROOT="$HOME/pi-tools/arm-bcm2708/arm-rpi-4.9.3-linux-gnueabihf/arm-linux-gnueabihf/sysroot";
+
 export CC="$TOOLCHAIN/gcc-sysroot";
 export AR="$TOOLCHAIN/arm-linux-gnueabihf-ar";
 

